src/main/java/ro/chirila/programarispital/controller/AppointmentController.java

```diff
@@ -0,0 +1,50 @@
+
+package ro.chirila.programarispital.controller;
+
+import jakarta.transaction.Transactional;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ro.chirila.programarispital.repository.dto.AppointmentRequestDTO;
+import ro.chirila.programarispital.repository.dto.AppointmentResponseDTO;
+import ro.chirila.programarispital.repository.dto.AppointmentUpdateDTO;
+import ro.chirila.programarispital.service.AppointmentService;
+import ro.chirila.programarispital.service.SendEmailService;
+
+import java.util.concurrent.CompletableFuture;
+
+@CrossOrigin(origins = "*", maxAge = 3600)
+@RestController
+@RequestMapping("/appointment")
+public class AppointmentController {
+
+    private final AppointmentService appointmentService;
+    private final SendEmailService sendEmailService;
+
+    public AppointmentController(AppointmentService appointmentService, SendEmailService sendEmailService) {
+        this.appointmentService = appointmentService;
+        this.sendEmailService = sendEmailService;
+    }
+    @PostMapping("/create-appointment")
+    @Transactional
+    public ResponseEntity<AppointmentResponseDTO> addAppointment(@RequestBody AppointmentRequestDTO appointmentRequestDTO
+            , @RequestParam String username){
+
+        AppointmentResponseDTO appointmentResponseDTO = appointmentService.addAppointment(appointmentRequestDTO,username);
+        CompletableFuture.runAsync(() -> sendEmailService.sendAppointmentEmail(appointmentResponseDTO));
+        return new ResponseEntity<>(appointmentResponseDTO, HttpStatus.OK);
```

Comment:
this should return HttpStatus CREATED instead of OK. See the list: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpStatus.html
Also, see that the other endpoints return the correct status.